translator/trace/big_endian_converter.go

```diff
@@ -102,3 +102,17 @@ func BytesToInt64SpanID(b []byte) (int64, error) {
 	id, err := BytesToUInt64SpanID(b)
 	return int64(id), err
 }
+
+// TraceIDBytesToUnsignedLowAndHigh converts 16byte TraceID to BigEndian format
+// and returns two uint64 integers as required by Jaeger format.
+func TraceIDBytesToUnsignedLowAndHigh(traceID []byte) (traceIDLow, traceIDHigh uint64) {
```

Comment:
@annanay25 instead of adding these functions could you please use the equivalent ones that also return error, e.g.: `BytesToUInt64TraceID `

Reply:
Sure, will do.

Reply:
Btw, @sjkaris suggested the error checking be moved up into the translate function - https://github.com/census-instrumentation/opencensus-service/pull/433#discussion_r261356395